spec/sql/billing/bill_cdr_batch_spec.rb

```diff
@@ -88,30 +70,58 @@
         term_gw_id: gw.id,
         duration: -11 # should be skipped in stats
       }
-    ].to_json.to_s
+    ]
   end
 
-  it 'responds with correct rows' do
+  it 'updates customer accounts' do
+    customer_acc1_balance = customer_acc1.balance
+    # destination_reverse_billing=false and customer_price > 0
+    customer_acc1_spending = batch_data[0][:customer_price] + batch_data[1][:customer_price]
+    # destination_reverse_billing=true and customer_price > 0
+    customer_acc1_income = batch_data[2][:customer_price]
+
+    customer_acc2_balance = customer_acc2.balance
+    # destination_reverse_billing=false and customer_price > 0
+    customer_acc2_spending = batch_data[4][:customer_price]
```

Comment:
calculate expected balance before calling `subject` with explanation.  
it's much more useful for other developers, than some magic numbers.  
such tests are easy to copy+paste into new contexts and change according to different test cases